pkg/web/apps/apps_http_handler.go

```diff
@@ -32,3 +32,14 @@ func (a *HTTPHandler) GetApps(c echo.Context) error {
 
 	return c.JSON(http.StatusOK, apps)
 }
+
+// GetAppsId returns apps by id as JSON from the AppService
+func (a *HTTPHandler) GetAppsID(c echo.Context) error {
+	apps, err := a.Service.GetAppsID(c)
+
+	if err != nil || apps == nil {
```

Comment:
Why an error should be return if the app not exist?

Reply:
to handle bad requests like 
/api/apps/2;DROP TABLE
will return 
```json
{
message: "Invalid id supplied",
statusCode: 400
}
```
where as 
/api/apps/2
will return
```json
{
message: "App not found",
statusCode: 404
}
```
and 
/api/apps/1
will return
```json
[
{
id: 1,
appId: "com.aerogear.app1",
appName: "app1",
deployedVersions: [
{
id: 1,
version: "com.aerogear.app1",
appId: "A1B2C3D4E5.com.domainname.applicationname",
disabled: false,
disabledMessage: "Disabled by admin",
numOfClients: 0,
numOfAppStartups: 0
}
]
}
]
```

Reply:
Please, check the other comments. When the data do not exist the HTTP code should not be 404.

Reply:
See that if the ID was informed as expected and if it is valid should be checked [here](https://github.com/aerogear/mobile-security-service/blob/7266e9ad0dc3e34b29038005842b60e1b99e79a4/pkg/web/apps/apps_http_handler.go) then the 400 returned.